api/src/models/user.ts

```diff
@@ -1,11 +1,86 @@
-import mongoose, { Document } from 'mongoose';
+import mongoose, { Document, Schema } from 'mongoose';
 
-interface User extends Document<any> {
+import { IPitch } from './pitch';
+
+/**
+ * Basic interface for a User Schema
+ */
+export interface IUser extends Document<any> {
   name: string;
+  preferredName: string;
+  email: string;
+  phone: string;
+  oauthID: string;
+  gender: string;
+  pronouns: string;
+  dateJoined: Date;
+  portfolio: string;
+  linkedIn: string;
+  twitter: string;
+  claimedPitches: [IPitch];
+  submittedPitches: [IPitch];
 }
 
+const interestsEnum = {
+  POLITICS: 'POLITICS',
+  EDUCATION: 'EDUCATION',
+  HOUSING: 'HOUSING',
+  LIT: 'LIT',
+  MUSIC: 'MUSIC',
+  VISUAL_ARTS: 'VISUAL ARTS',
+  STAGE_AND_SCREEN: 'STAGE AND SCREEN',
+  FOOD_AND_LAND: 'FOOD AND LAND',
+  NATURE: 'NATURE',
+  TRANSPORTATION: 'TRANSPORTATION',
+  HEALTH: 'HEALTH',
+  CANNABIS: 'CANNABIS',
+  IMMIGRATION: 'IMMIGRATION',
+  FUN: 'FUN',
+  NONE: 'NONE',
+};
+
+const rolesEnum = {
+  CONTRIBUTOR: 'CONTRIBUTOR',
+  STAFF: 'STAFF',
+  ADMIN: 'ADMIN',
+  TBD: 'TBD',
+};
+
+/**
+ * Mongoose Schema to represent a User at South Side Weekly
+ */
 const User = new mongoose.Schema({
-  name: { type: String },
+  firstName: { type: String, default: null, required: true },
+  middleName: { type: String, default: null },
+  lastName: { type: String, default: null, required: true },
+  preferredName: { type: String, default: null },
+  email: { type: String, default: null },
+  phone: { type: String, default: null },
+  oauthID: { type: String, default: null, unique: true, sparse: true },
+  gender: [{ type: String, default: null }],
+  pronouns: [{ type: String, default: null }],
+  race: [{ type: String, default: null }],
```

Comment:
Change this to a string enum, you can view the options they currently have in the race section of the [Become a Contributor form](https://southsideweekly.com/contribute/)